x-pack/plugins/secops/server/lib/ecs_fields/index.ts

```diff
@@ -60,11 +60,18 @@ export const suricataFieldsMap: Readonly<Record<string, string>> = {
 export const sourceFieldsMap: Readonly<Record<string, string>> = {
   'source.ip': 'source.ip',
   'source.port': 'source.port',
+  'source.domain': 'source.domain',
 };
 
 export const destinationFieldsMap: Readonly<Record<string, string>> = {
   'destination.ip': 'destination.ip',
   'destination.port': 'destination.port',
+  'destination.domain': 'destination.domain',
+};
```

Comment:
Optional: I would go ahead and add the other fields while you're in here:
https://github.com/elastic/ecs#-source-fields

But up to you. Someone will soon enough I'm sure.